virtme_ng/version.py

```diff
@@ -37,9 +48,9 @@ def get_version_string():
 
         return version_pep440
     except CalledProcessError:
-        # Default version if git describe fails (e.g., when building virtme-ng
-        # from a source package.
-        return PKG_VERSION
+        # If git describe fails to determine a version, try to use the version
+        # from pip, or ultimately simply return the hard-coded package version.
+        return get_package_version()
 
 
 VERSION = get_version_string()
```

Comment:
(not related to this PR, and it's a detail, but if the version is only used when requested, maybe it would be better not to export `VERSION`, only `get_version_string`, not to query git each time vng is used :) )

(and maybe by doing that, arguments could be passed to `get_version_string()` instead of relying on `__VNG_LOCAL`, but again, detail, feel free to ignore :) )

Reply:
I was also thinking about that and honestly I don't see any downside, but I need to do some tests. Will work on that, thanks for the review!